sktime/datatypes/_hierarchical/_registry.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pd
+
+__all__ = [
+    "MTYPE_REGISTER_HIERARCHICAL",
+    "MTYPE_LIST_HIERARCHICAL",
+]
+
+
+MTYPE_REGISTER_HIERARCHICAL = [
+    (
+        "pd_multiindex_hier",
+        "Hierarchical",
+        "pd.DataFrame with MultiIndex",
+    ),
```

Comment:
Is it a good idea to mix dots and underscores with naming (and capitals) I'd suggest all lower and underscores.

Reply:
yes, exactly.

I'd also like to move to all lower case and underscores, but we can't easily change this for the currently existing mtypes/scitypes, it would require deprecation.

But we can changei t for the new ones, that's why I used all lower case and underscore which is inconsistent with `pd-multiindex` (which I'd like to rename to `pd_multiindex` later).

Reply:
but there wouldn't be an action here, right? the third string is not a name, but an English description of the mtype.